lib/util/regex.ts

```diff
@@ -98,3 +98,12 @@ export function configRegexPredicate(
   }
   return null;
 }
+
+const UUIDRegex = regEx(
+  /^\{[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}\}$/,
+  'i'
+);
+
+export function checkUUID(input: string): boolean {
```

Comment:
```suggestion
export function isUUID(input: string): boolean {
```
`is` communicates that we will return a boolean.